cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -7,7 +7,15 @@ class HttpCache
 
   # Language header and cookie are needed to separately cache language-specific pages.
   LANGUAGE_HEADER = %w(Accept-Language)
-  LANGUAGE_COOKIES = %w(language_ pm)
+
+  DEFAULT_COOKIES = [
+    # Language drop-down selection.
+    'language_',
```

Comment:
We had some caching issues before when we were on a page that wasn't supposed to be localized but contained a view that was localized. [Example of a page](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/3rdparty-partners.md) that contains the share_buttons view that is localized. Or code.org/volunteer where we started to localize the fields but never enabled localization on the page. And we had caught different languages popping up.

Is this what you mean? Will fixed those issues.